config/kubernetes/sig-docs/teams.yaml

```diff
@@ -18,31 +18,44 @@ teams:
     members:
     - bradamant3
     - bradtopol
-    - chenopis
-    - cody-clark
+    - daminisatya
     - gochist
     - jaredbhatti
     - jimangel
     - kbarnard10
     - kbhawkey
     - makoscafee
-    - mistyhacks
     - Rajakavitha1
     - ryanmcginnis
     - simplytunde
     - steveperry-53
     - stewart-yu
-    - tfogo
+    - tengqm
     - xiangpengzhao
     - zacharysarah
     - zparnold
     privacy: closed
   sig-docs-en-reviews:
     description: PR reviews for English content
     members:
+    - bradamant3
+    - bradtopol
+    - daminisatya
+    - gochist
+    - jaredbhatti
+    - jimangel
+    - kbarnard10
+    - kbhawkey
+    - makoscafee
     - Rajakavitha1
-    - sftim
```

Comment:
@zacharysarah, if you wish: you could leave me in this list and add me to owners, and then close #1525